pkg/ebpf/c/tracee.bpf.c

```diff
@@ -981,6 +989,7 @@ int uprobe_lkm_seeker_submitter(struct pt_regs *ctx)
 
     // Uprobes are not triggered by syscalls, so we need to override the false value.
     p.event->context.syscall = NO_SYSCALL;
+    p.event->context.policies_version = p.config->policies_version;
```

Comment:
Regarding this, I'll wait @OriGlassman @AlonZivony feedback.

Reply:
> Because note that there can be a policy version change in the time between the uprobe trigger and the ebpf program starting.

I had put a check on the last stage of the event submission, checking the event context version against the current one from config_entry; if different discard the event. We had discussions about that and agreed that it would be better to send the event even with wrong version. But we could reintroduce it again to set the version as 0, so userland would know that it's a rogue event.

Reply:
> checking the event context version against the current one from config_entry;
But even the event context version can be later than the one that should be injected from the trigger. So that kind of corruption would be missed. This isn't a huge problem for our trigger events right now since they are infrequent, but if we had events which are more frequent, it may be more noticable.
I'd at least document the limitation, it may not be worth fixing (I think eBPF cookies in later kernels could be used to solve this for example).